com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -560,7 +560,7 @@ public static List<CommandLineOption> getCommandLineOptions() {
                 opt(null, "with-symbols", ZERO, "Generate the symbol file (if applicable)", false),
 
                 opt("tp", "texture-profiles", ONE, "DEPRECATED! Use --texture-compression instead", true),
-                opt("tc", "texture-compression", ONE, "Use texture compression as specified in texture profiles", true),
+                opt("tc", "texture-compression", ZERO, "Use texture compression as specified in texture profiles", true),
```

Comment:
What happens with user scripts passing an argument to this property? Breaking change?
User facing documentation?